### ISSUE TITLE
MAE upgrade breaks back end Applications pages

### ISSUE THREAD

**Issue by rogerdavis198** [Xtraball/Siberian#828]
MAE upgrade from 501 breaks the backoffice app info and all the data and make apk etc in the Applications pages

_####(V4 v5) also only the banner ads seem to work if your lucky as even those tend to not work on some apps. Please fix as I am stuck on v4_#######

 ALL ADS WORK OK NOW

I am also noticing that that when you put ads in a app it throws the logo to one side ?? So it not central.in the app anymore php.7.33 .8.2 SAE

V5 front end Php 8.2 is worse and loses pages on front end in some apps too.

Thanks guys

Done lots of test upgrades and it never upgrades back end all the way to 509

its loosing all the separate app info and make apk aab etc will post picture of the problem

**Comment by rogerdavis198** [comment]
Note SAE is also messing the template up , with the task tracker template , I started off with 3 boxes across and after an update it is only 2 boxes and all.is on the left now !!

**Comment by rogerdavis198** [comment]
![17023031824556306736630893664485](https://github.com/Xtraball/Siberian/assets/24929628/6094845a-333d-4975-b7c5-8300d2293003)
![Uploading 17023032074837472363678714967197.jpg…]()

**Comment by rogerdavis198** [comment]
![17023033260851814709367149671923](https://github.com/Xtraball/Siberian/assets/24929628/319bd6e1-0fc1-47d3-8336-0d4645a8ac58)
As you can see only 2 rows and the banner ads have disappeared too

**Comment by rogerdavis198** [comment]
There has to be a bug in this software as its doing the same as the radio players in different formats ...
If you use the html site of the app
All radio players play in ogg opus aac etc

If you use html site of above app it's back to 3 blocks wide but the ads are still missing. Is it a memory lock ?? Causing all these problem ...

**Comment by rogerdavis198** [comment]
![17023043648025351447337094094300](https://github.com/Xtraball/Siberian/assets/24929628/60327337-5a73-42a3-a2f7-261963ffffac)
Same app in html site only

**Comment by rogerdavis198** [comment]
![17025569336286965705807474408830](https://github.com/Xtraball/Siberian/assets/24929628/801bfe38-db87-475d-a81e-c99c812eb32e)

![1702556985858728340208923636095](https://github.com/Xtraball/Siberian/assets/24929628/6262687d-f722-4458-a18b-5fb3f3a081c3)
![Uploading 17025570111138595683344478387576.jpg…]()

**Comment by rogerdavis198** [comment]
As you can see all is blank For every app

**Comment by rogerdavis198** [comment]
![Uploading 17025573568615442175498236693748.jpg…]()

**Comment by rogerdavis198** [comment]
The last pictures should show that the sourced tab is also blank so no generated sources button,. You can from the fronted,  Been same from v5 to v509

**Comment by Slicit** [comment]
@rogerdavis198 as said earlier, if you have lot of modules better STAY on the supported PHP 7.3

PHP 8.2 is early support and only Siberian Core & Siberian modules are known to be up-to-date

Blank pages/content is generally due to such code not working

**Comment by rogerdavis198** [comment]
Yes php 8.2 understand that but the above is v5 with PHP version:  7.3.33 only
cheers
Roger

**Comment by Slicit** [comment]
So try a restore app sources or something else

And if no luck, put in development and copy logs of php errors ;)

**Comment by rogerdavis198** [comment]
I just running permission fix on that server but v4 is perfect before any upgrades to v5, will dev it and check errors

**Comment by rogerdavis198** [comment]
php 733 I put it in dev and there is no errors at all on any of the apps, there is one template as listed earlier if on full goes all over the place and backoffice only seems to half work and does not talk to the database by looks of it,you can not clear the locks on it too, i have run the crons from the server end and i still have it says in backoffice 1.4 gigs of temp files. going to fire my v4 server up as thats works great at the mo,
